.github/workflows/pr_local_integration_tests.yml

```diff
@@ -77,4 +77,8 @@ jobs:
         run: gcloud info
       - name: Test local integration tests
         if: ${{ always() }}  # this will guarantee that step won't be canceled and resources won't leak
-        run: make test-python-integration-local
+        env:
+          FEAST_USAGE: "False"
+          IS_TEST: "True"
+          FEAST_LOCAL_ONLINE_CONTAINER: "True"
+        run: pytest -n 8 --cov=./ --cov-report=xml --color=yes --integration -k "not test_apply_entity_integration and not test_apply_feature_view_integration and not test_apply_data_source_integration and not test_lambda_materialization" sdk/python/tests
```

Comment:
There was a bug where pr_local_integration_test isn't actually failing even if the workflow was failing

Reply:
this shouldn't be relevant anymore right? you should be able to revert